.ci/docker/requirements.txt

```diff
@@ -1,6 +1,7 @@
 torchdata >= 0.8.0
 datasets >= 2.21.0
 tomli >= 1.1.0 ; python_version < "3.11"
+tomli-w >= 1.1.0 ; python_version < "3.11"
```

Comment:
Can we put this into `dev-requirements.txt`, instead of `requirements.txt`
I checked it will be picked up by CI https://github.com/pytorch/torchtitan/blob/main/.ci/docker/common/install_conda.sh#L42

Reply:
Hmm, I made the change and it seems to have fixed CI. I didn't understand why though, is .ci/requirements.txt not being used?